platform.py

```
@@ -522,6 +526,17 @@ def install_tool(self, tool_name: str) -> bool:
                 not status['has_tools_json']):
             return self._handle_existing_tool(tool_name, paths)
 
+        # Case 3: Package not yet downloaded – fetch skeleton then install
```

Ruff (0.15.2)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)